.travis.yml

```diff
@@ -1,14 +0,0 @@
-os:
```

Comment:
It'd be better to keep this file as it is used by Travis CI to run TSlint and pbiviz package for each PR that we receive.